spikeinterface/toolkit/postprocessing/correlograms.py

```diff
@@ -22,14 +162,16 @@ def compute_correlograms(sorting,
 
     fs = sorting.get_sampling_frequency()
 
-    window_size = int(fs * window_ms / 1000.)
-    bin_size = int(fs * bin_ms / 1000.)
+    window_size = int(round(fs * window_ms / 2000.))
+    bin_size = int(round(fs * bin_ms / 1000.))
+    window_size -= window_size % bin_size
+    window_size *= 2
     real_bin_duration_ms = bin_size / fs * 1000.
 
     # force odd
-    num_total_bins = 2 * int(.5 * window_size / bin_size) + 1
+    num_total_bins = window_size // bin_size
```

Comment:
@Dradeliomecus this is not forcing to an odd number of bins, right? Isn't it better to have an odd number so that there is a central bin and then the correlogram is symmetric?

Reply:
Ah maybe...
I wrote everything for an even number of bins (so there would bin the bin `[0 ; bin_size[`).
If you want an odd number (so that there is one bin centered on 0), then you need an odd `bin_size`, and I didn't think this restriction was worth it.

If you want this however, I can do it.